protected/routers/cli.php

```diff
@@ -0,0 +1,43 @@
+<?php
+/**
+ * Trait For Routing HTTP Requests
+ */
+trait Router
+{
+    /**
+     * Perform Redirect Rules
+     *
+     * @param string $uri HTTP Request URI
+     */
+    public function routeRedirect(?string $uri = null): void
+    {
+        if (empty($uri)) {
+            $uri = '/';
+        }
+
+        if (!preg_match('/^\/$/su', $uri)) {
+            header('Location: /');
+            exit(0);
+        }
+    }
+
+    /**
+     * Perform Rewrite Rules
+     *
+     * @param string $uri HTTP Request URI
+     *
+     * @return string Rewrited HTTP Request URI
+     */
+    public function routeRewrite(?string $uri = null): string
```

Comment:
Codacy found an issue: [Method name "routeRewrite" in class Router is not in snake case format, try "route_rewrite"](https://app.codacy.com/manual/common/buryak/pullRequest?prid=5800482)